ansible/roles/destroy-vm/tasks/main.yml

```diff
@@ -15,3 +15,7 @@
 - name: Delete IBM Z VM
   include_tasks: destroy-s390x-vm.yml
   when: arch_info == "s390x"
+
+- name: Delete IBM POWER VSI
+  include_tasks: destroy-ppc64le-vm.yml
```

Comment:
If we can avoid getting the hostname of the VM, these tasks can (and should) be run on localhost - so we can do:

```
- name: Delete IBM Power VSI
  include_tasks: 
      file: destroy-ppc64le-vm.yml
      apply:
        delegate_to: localhost
  when: arch_info == "ppc64le"
```

which will mean we can remove all the lines in `destroy-ppc64le-vm.yml` that delegate to localhost, and just enforce it at a higher level

Reply:
I tried this but it is somehow not applying 
ansible 2.7 supports this apply functionality

Reply:
This is fine for now I think. Most important is removing the need to get the VM hostname for deletion